2006Scape Client/src/main/java/ObjectManager.java

```diff
@@ -1355,6 +1355,10 @@ public final void method190(int i, CollisionMap aclass11[], int j, WorldControll
 	private final byte[][][] aByteArrayArrayArray148;
 	private final byte[][][] aByteArrayArrayArray149;
 	static boolean lowMem = true;
+	
+	static boolean hideRoofs = ClientSettings.HIDE_ROOFS;
+	
+	static boolean hideBuggyVarrockSwordShopSnow = true; //There is some buggy snow near Varrock sword shop, let's hide it. I know, this is not ideal, but it does work.
```

Comment:
this should be in ClientSettings.java i'd say

Reply:
Agreed.